cmds/create_translations.go

```diff
@@ -90,7 +85,8 @@ func NewCreateTranslationsCommand(options *common.Options) *cobra.Command {
 	createTranslationsCmd.Flags().StringVar(&options.GoogleTranslateApiKeyFlag, "google-translate-api-key", "", "[optional] your public Google Translate API key which is used to generate translations (charge is applicable)")
 	createTranslationsCmd.Flags().StringVarP(&options.SourceLanguageFlag, "source-language", "s", "en", "the source language of the file, typically also part of the file name, e.g., \"en_US\"")
 	createTranslationsCmd.Flags().StringVarP(&options.FilenameFlag, "file", "f", "", "the source translation file")
-	createTranslationsCmd.Flags().StringSliceVarP(&options.LanguagesArrayFlag, "languages", "l", []string{}, "a comma separated list of valid languages with optional territory, e.g., \"en, en_US, fr_FR, es\"")
+	createTranslationsCmd.Flags().StringVarP(&options.LanguagesFlag, "languages", "l", "", "a comma separated list of valid languages with optional territory, e.g., \"en, en_US, fr_FR, es\"")
```

Comment:
**Callout:** Cobra interprets a quoted list (eg. `"a,b,c"`) as a  string when we expect an array. To allow flexibility, I have changed it to a string and have the CLI parse the string